docs/glossary.md

```diff
@@ -27,6 +27,7 @@
 | LRN               | Local Response Normalization |
 | mAP               | Mean Average Precision |
 | Intel(R) MKL-DNN  | Intel(R) Math Kernel Library Deep Neural Networks |
+| Intel(R) OneDNN   | Intel(R) OneAPI Deep Neural Network Library |
```

Comment:
Perhaps the MKL-DNN description should be removed?

Reply:
removed